docs/wallets/ledger-wallet.mdx

```diff
@@ -19,4 +19,215 @@ for:
 - Transferring tokens
 - Signing transactions
 
+:::important
```

Comment:
This is no longer true, you should be able to use a ledger to onboard/assert location in the wallet app

Reply:
Thanks for pointing out, already resolved!